### DISCUSSION TITLE
error when adding datacube product 'Unknown metadata type: 'eo3_landsat_ard''

### DISCUSSION THREAD

**Discussion by Zina-Kamel** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#58] [1.0 Getting started on the Data Challenge Platform - https://datascience.ey.com/]
I am using my local environment to get started. I have followed the instructions provided on GitHub and have set up the environment correctly. However, in the final step of step 3 ( “Now index some datasets”) I am getting this error: ValueError: Supplied product name "ga_ls7e_ard_3" not present in the database 

I have cloned the GitHub repository correctly and my docker container is running. However, when I try to add `ga_ls7e_ard_3.odc-product.yaml` or `ga_ls8c_ard_3.odc-product.yaml`  products I am getting the following error ` raise InvalidDocException('Unknown metadata type: %r' % definition['metadata_type'])
datacube.utils.documents.InvalidDocException: Unknown metadata type: 'eo3_landsat_ard'
 datacube-product ERROR Invalid product definition: /scripts/data/ga_ls7e_ard_3.odc-product.yaml`

The other products (such as the linescan) are working perfectly and I have indexed the datasets for them
I also tried to redownload the ls78.tar.gz and ga_ls7e_ard_3.odc-product but it is still giving me the same error

**Comment by EY-glowglowglow** [comment]
@alexgleith  @codeindulgence any ideas on how to resolve this?

**Comment by Zina-Kamel** [comment]
Hello again!
A little update: I was able to resolve the issue by manually redownloading the data from github as a zipped file. I guess there was a problem with one of the files I had. I hope this helps someone having a similar problem in the future!

**Reply by nickbutler** [reply]
Hi @Zina-Kamel, glad you were able to resolve this!
